apps/social/src/App.tsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect } from 'react';
+import { useApi, useBalanceSubscription, useLoggedInAccount } from '@gear-js/react-hooks';
+import { Routing } from 'pages';
+import { Header, Footer, ApiLoader } from 'components';
+import { withProviders } from 'hocs';
+import { LOCAL_STORAGE } from './consts';
+import 'App.scss';
+
+function Component() {
+  const { isApiReady, api } = useApi();
+
+  // Set Genesis block
+  useEffect(() => {
+    if (isApiReady) {
+      localStorage.setItem(LOCAL_STORAGE.GENESIS, api.genesisHash.toHex());
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [isApiReady]);
```

Comment:
I don't see any reason to store genesis in local storage, since you only use it inside hooks where you can still get it with `api.genesisHash`

Reply:
ok